examples/tensorflow/train/crnn/README.md

```diff
@@ -0,0 +1,106 @@
+
+If you want to train your own datasets,you just need to storage you datasets in the same directory as the Test and Train folders and write relative path and labels of image data in Test/sample.txt and Traing/sample.txt.
+ Char_dict.json and ord_map.json must be placed in the /data/data folder . All your training image will be scaled into (32, 100, 3) , the dataset will be divided into train, test, validation set and you can change the parameter to control the ratio of them.
+ If you want to use your dataset to train CRNN net , there should be only one line character in each of your images.<br>
+
+
+
+```
+/_ data/
+  |_ data/
+	 |_ char_dict.json
+	 |_ ord_map.json
+     |_ Test/
+        |_ sample.txt
+     |_ Train/
+        |_ sample.txt
+     |_ 1/
+        |_ 2/
+		   |_ 373_coley_14845.jpg
+           |_ 100_eunice_26759.jpg
+     |_ ...
+  |_ code/
+     |_ crnnmodel/
+	 |_ data_provoder/
+	 |_ global_configuration/
+	 |_ local_utils/
+	 |_ tools/
+  |_ output/
+  |_ crnn-generate-tfrecords.Dockerfile
+```
+
+You can run the following command to generate a docker image named uhub.service.ucloud.cn/uai_demo/crnn-generate-tfrecords:v1.0 to generate tfrecords file.You should change 'uai_demo' to 'YOUR_UHUB_REGISTRY'.
+```
+sudo docker build -t uhub.service.ucloud.cn/uai_demo/crnn-generate-tfrecords:v1.0 -f crnn-generate-tfrecords.Dockerfile .
+```
+In the docker image,you can generate tfrecords file locally or on UAI Training Platform .<br>
+If you want to generate tfrecords file locally , you should map ./data and ./code to crnn-generate-tfrecords:v1.0 by the following command.
+
+```
+sudo docker run -it -v /data/data:/data/data -v /data/output:/data/output uhub.service.ucloud.cn/uai_demo/crnn-generate-tfrecords:v1.0
+```
+Then you can run the following command to generate tfrecords file locally or on UAI Training Platform .<br>
+```
+python /data/code/tools/write_text_features.py --dataset_dir /data/data/ --save_dir /data/output
+```
+If you run the command locally,you can get train_feature.tfrecords,test_feature.tfrecords and validation_feature.tfrecords in /data/output folder.
+If you generate tfrecords file on UAI Training Platform,you can find tfrecords file on Ucloud file storage platform such as UFile and UFS.
+
+## Training model 
+Firstly,you can storage data according to the following structure. 
+```
+|_ data/
+  |_ data/
+     |_ train_feature.tfrecords
+	 |_ test_feature.tfrecords
+	 |_ validation_feature.tfrecords
+	 |_ char_dict.json
+	 |_ ord_map.json
+  |_ code/
+     |_ crnnmodel/
+	 |_ data_provoder/
+	 |_ global_configuration/
+	 |_ local_utils/
+	 |_ tools/
+  |_ ocr-cpu.Dockerfile 
+```
+You can build docker image by Dockerfile.
+
+```
+sudo docker build -t uhub.service.ucloud.cn/uai_demo/ocr_test:v1.0 -f ocr-cpu.Dockerfile .
+```
+So you can get a docker image named uhub.service.ucloud.cn/uai_demo/ocr_test:v1.0.You should change 'uai_demo' to 'YOUR_UHUB_REGISTRY'.<br>
```

Comment:
You should change 'uai_demo' to 'YOUR_UHUB_REGISTRY' to build your own docker image.